src/VerifyCustomFactorController.js

```diff
@@ -71,7 +84,11 @@ export default FormController.extend({
     });
     const vendorName = factor.get('vendorName');
     const saveText = loc('mfa.challenge.verify', 'login');
-    const subtitle = loc('verify.customFactor.subtitle', 'login', [vendorName]);
+    let subtitle = loc('verify.customFactor.subtitle', 'login', [vendorName]);
+
+    if (this.settings.get('features.skipIdpFactorVerificationBtn')) {
+      subtitle = loc('verify.customFactor.subtitle.redirect', 'login', [vendorName]);
```

Comment:
I actually think we need both "subtitles", the reason is, when we redirect we show "redirecting to..." but if there is an error, we should show the button and the subtitle saying "Clicking below will redirect to ..." so the user can try again if needed (you can see on the screenshot you have on this PR that the error message reads "review the form and make corrections", which is not ideal for this screen, but in any case we should give the user a way to retry. Feel free to confirm with UX)